tests/serializers/test_location.py

```diff
@@ -82,14 +82,14 @@ def test_validate_bad_data(self):
             _fill_optional=['shapefile'])
         missing_radius = OrderedDict(
             name='Nairobi',
-            geopoint='Point(0.0, 0.0)')
+            geopoint='POINT(30 10)')
```

Comment:
why are we removing the comma?

Reply:
Point(0.0, 0.0) was not an actual WKT for the GeoDjango Model and threw some errors around. POINT(30 10) on the other hand is a valid WKT for GeoDjango

Reply:
Cool